applications/TopologyOptimizationApplication/tests/Small_Cantilever/run_TopOpt.py

```diff
@@ -0,0 +1,124 @@
+#import kratos core and applications
+import KratosMultiphysics as km
+import KratosMultiphysics.StructuralMechanicsApplication as ksm
+import KratosMultiphysics.TopologyOptimizationApplication as kto
```

Comment:
Please, use the analysis as it is done in the other apps

Reply:
Yes I agree this script structure is very old

=> check the AnalysisStage in the Core and the StructuralMechanicsAnalysis for example

Reply:
I would do that in a second moment if possible.